exp/topology/scope/hookresponsetracker_test.go

```diff
@@ -164,7 +164,7 @@ func TestHookResponseTracker_IsBlocking(t *testing.T) {
 		},
 	}
 
-	afterClusterUpgradeResponse := &runtimehooksv1.AfterClusterUpgradeResponse{
+	afterClusterUpgradeResponse := &runtimehooksv1.AfterControlPlaneInitializedResponse{
```

Comment:
Let's also rename the variable and also adjust the godoc below (grep for AfterClusterUpgrade)